device/rsvp/rsvpFastReroute.yaml

```diff
@@ -0,0 +1,75 @@
+components:
+  schemas:
+    Rsvp.FastReroute:
+      description: >-
+        Configuration for the optional RSVP-TE FAST_REROUTE object included in Path Messages as defined in RFC4090. 
+      type: object
+      properties:
+        setup_priority:
+          description: >-
+            Specifies the value of the Setup Priority field. This controls whether the backup LSP should pre-empt existing 
+            LSP setup with certain Holding Priority if resource limitation is encountered when setting up the LSP. (e.g bandwidth availability)
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        holding_priority:
+          description: >-
+            Specifies the value of the Holding Priority field. This controls whether a new LSP being created with certain Setup Priority
+            should pre-empt this LSP if resource limitation is encountered when setting up the LSP. (e.g bandwidth availability)
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        hop_limit:
+          description: >-
+            Specifies the value of the Hop Limit field. This controls the maximum number of hops the LSP should traverse to reach the 
+            LSP end-point.
+          type: integer
+          default: 3
+          minimum: 0
+          maximum: 255
+        bandwidth:
+          description: >-
+            Specifies the value of the Bandwidth field in bytes per second desired for the LSP. 
```

Comment:
Specifies the value of the Bandwidth field in bytes per second (mbps) desired for the LSP.

Reply:
Left out bps ( not mbps ) since sometimes it is interpreted as bits per second. bytes per second seems unambiguous without any need for futher clarification.